labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang="en-US">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>Lab 1 - Chirpus Site</title>
+  <link rel="stylesheet" type="text/css" href="./style.css">
+  <link rel='stylesheet' media='screen and (min-width: 100px) and (max-width: 900px)' href="./mobileStyle.css"/>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800" rel="stylesheet">
+</head>
+<body>
+<main>
+  <header class="banner">
+    <p>Chirpus Inc.<p>
+    <div class="middle-space"></div>
+    <nav class="nav-list">
```

Comment:
Love the `<nav>` `<ul>` combination; but in general you'll only have one `nav` on the page, so no need for a class. Just target `nav` directly unless there's a reason not to.